nam/models/base.py

```diff
@@ -259,3 +275,20 @@ def _mse_loss(self, preds, targets, pre_emph_coef: Optional[float] = None):
                 z[..., 1:] - pre_emph_coef * z[..., :-1] for z in (preds, targets)
             ]
         return nn.MSELoss()(preds, targets)
+
+    def _mrstft_loss(self, preds: torch.Tensor, targets: torch.Tensor) -> torch.Tensor:
+        """
+        Experimental Multi Resolution Short Time Fourier Transform Loss using auraloss implementation.
+        B: Batch size
+        L: Sequence length
+
+        :param preds: (B,L)
+        :param targets: (B,L)
+        :return: ()
+        """
+        device = 'cpu' # not all platforms support this on gpu yet
```

Comment:
> `# not all platforms support this on gpu yet`

Ooh really? Is there a custom kernel that's needed?

What's the performance hit (i.e. iteration speed) look like for having to do this? (I assume not the end of the world and I don't think I'd reject the PR over it; just curious)

Reply:
Did not work in apple metal GPU (uses unimplemented methods), have not tried yet in nvidia.